### PR TITLE
Contribution guide for litmus-docs

### PR DESCRIPTION
**Pull request by k8s-dev**
This commit has process about getting started to contribute towards
litmus-docs repo.

Fixes # https://github.com/litmuschaos/litmus/issues/1942 
Signed-off-by: Jayesh Kumar Tank <k8s.cncf@gmail.com>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Welcome to Litmus!! This guide goes over in details about the contribution process to the documentation.
+
+Litmus is an Apache 2.0 Licensed project and uses the standard GitHub pull requests process to review and accept contributions. We welcomes improvements from all contributors, new and experienced!
+
+Litmus documentation contributors:
+
+* Improve existing content
+* Create new content
+* Manage and publish the documentation parts of the Litmus release cycle
```
Comment (ksatchit) [inline CONTRIBUTING.md]: docs framework, build process.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Contributions are not limited to the above mentioned items. In case you have ideas for improving the documentation - create a PR.
+
+
+## Steps to Contribute
+Litmus documentation currently runs using Docusaurus static file generator. Travis CI is the build tool. Also, we have integration with Netlify to deploy preview of the change.
+
+* Find an issue to work on or create a new issue. The issues are maintained at [litmuschaos/litmus](https://github.com/litmuschaos/litmus/issues). For new contributors, you can pick up from a list of [good first issues](https://github.com/litmuschaos/litmus/labels/good%20first%20issue)
```
Comment (ksatchit) [inline CONTRIBUTING.md]: area/litmus-docs

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+* Claim your issue by commenting your intent to work on it to avoid duplication of efforts. 
+* Fork the repository on GitHub.
+* Create a branch from where you want to base your work (usually master).
```
Comment (ksatchit) [inline CONTRIBUTING.md:24]: Ref: Standard git OSS procedure (link)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+* Make your changes, test locally, see preview by starting server and seeing in Preview Mode.
```
Comment (ksatchit) [inline CONTRIBUTING.md:25]: Add step to view preview (comment w/ preview link) and escalate if preview fails.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+* Commit your changes by making sure the commit messages convey the need and notes about the commit.
```
Comment (ksatchit) [inline CONTRIBUTING.md:27]: Pull the latest version of the kubernetes snippet being used in the doc by using embedmd tool. Ref: https://github.com/litmuschaos/litmus-docs#use-embedmd-command-before-commiting-changes

Comment (ksatchit) [reply CONTRIBUTING.md:27]: Ref: http://github.com/campoy/embedmd/

File: CONTRIBUTING.md (hunk 2 above)
Comment (ksatchit) [reply CONTRIBUTING.md]: https://github.com/litmuschaos/litmus/issues?q=is%3Aopen+label%3A%22good+first+issue%22+label%3A%22area%2Flitmus-docs%22

File: CONTRIBUTING.md (hunk 5 above)
Comment (ksatchit) [reply CONTRIBUTING.md:27]: @k8s-dev can we add this too?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
+# Contributing to Litmus Documentation
+
+Welcome to Litmus!! This guide goes over in details about the contribution process to the documentation.
+
+Litmus is an Apache 2.0 Licensed project and uses the standard GitHub pull requests process to review and accept contributions. We welcomes improvements from all contributors, new and experienced!
```
Comment (divya-mohan0209) [inline CONTRIBUTING.md]: Minor grammatical edit: "We welcome improvements from all contributors, new and experienced."

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+Litmus documentation contributors:
+
+* Improve existing content
+* Create new content
+* Maintain documentation framework
+* Keep documentation builds healthy
+* Manage and publish the documentation parts of the Litmus release cycle
```
Comment (divya-mohan0209) [inline CONTRIBUTING.md]: Possible rewrite to: Manage & publish documentation as part of the Litmus release cycle.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Minor grammatical + spelling changes. Also, would look at standardizing the way Litmus is capitalized across the documentation (barring links).

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+We organize the Litmus into Special Interest Groups or SIGs in order to improve our workflow and to easily manage this community project. The developers within each SIG have autonomy and ownership over that SIG’s part of Litmus.
+
+Like everything else in Litmus, a SIG is an open, community, effort. We thrive on a lively and friendly open-source community. Anybody is welcome to jump into a SIG and begin fixing issues/documentation, critiquing design proposals and reviewing code.
+
+The litmus community will have a weekly contributor sync-up on Mondays 20.30-21.30IST 
+- The sync up meeting is held online on [zoom](https://zoom.us/j/91358162694)
+- The contribution items are tracked in this [planning sheet](https://docs.google.com/document/d/1Z9DrnA8W_IM2HnVOWU1dtVrVB9MyWj8VP43YS_A_wFs).
+
+Documentation SIG lives [here](https://github.com/litmuschaos/litmus/wiki/Special-Interest-Groups#sig-documentation). Please join the weekly zoom call, introduce yourself and get started!!
```
Comment (akria18) [inline CONTRIBUTING.md]: SIG documentations are here

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+* Create new content
+* Maintain documentation framework
+* Keep documentation builds healthy
+* Manage and publish the documentation parts of the Litmus release cycle
+
+Contributions are not limited to the above mentioned items. In case you have ideas for improving the documentation - create a PR.
+
+
+## Steps to Contribute
+Litmus documentation currently runs using [Docusaurus](https://docusaurus.io/) static file generator. [Embedmd](https://github.com/campoy/embedmd/) for embedding source codes into the documentation. [Travis CI](https://travis-ci.org/) is the build tool. Also, we have integration with [Netlify](https://www.netlify.com/) to deploy preview of the change.
```
Comment (akria18) [inline CONTRIBUTING.md]: Also, we have integrated Netlify to deploy preview of the change.

**Review by akria18 — DISMISSED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**